JumpscaleCore/data/bcdb/BCDB.py

```diff
@@ -50,7 +50,7 @@ def _init(self, name=None, storclient=None, reset=False, readonly=False):
         self.storclient = storclient
 
         j.sal.fs.createDir(self._data_dir)
-        self.readonly = readonly
+        self.readonly = not j.data.bcdb._master
```

Comment:
let's remove readonly from constructor

Reply:
it changes the function signature, I really don't want us to start refactoring at this point